src/main/java/nl/tudelft/ti2206/group9/level/State.java

```diff
@@ -25,22 +19,13 @@
     /** Highest score ever obtained. */
     private static double highscore;
 
-    /** Skin to be used. */
-    private static Skin skin;
-
     /** Current track, contains all entities. */
-    private static Track track = new Track();
+    private static Track track;
```

Comment:
Is this field still used in State? It should be removed as Track is a Singleton now.

<review, logic, interface